hvpy/io.py

```diff
@@ -54,4 +56,24 @@ def url(self) -> str:
         """
         Final API endpoint URL.
         """
-        return BASE_URL + self.__class__.__name__[:-15] + "/"
+
+        return get_api_url() + self.__class__.__name__[:-15] + "/"
+
+
+def get_api_url() -> str:
+    """
+    Returns the base API URL.
+    """
+    if _base_url:
+        return _base_url
+    if "PRIVATE_URL" in os.environ:
```

Comment:
This variable name is still too vague.

Reply:
Yes, need to change PRIVATE_URL to something more helioviewer specific.